dash/frontend/src/app/modules/private/pages/image/image-scan-result/image-scan-result.component.ts

```diff
@@ -146,7 +146,7 @@ export class ImageScanResultComponent implements OnInit, AfterViewInit, OnDestro
       .subscribe((namespaces) => {
           this.imageNamespaces = namespaces.data.map(namespace => namespace.namespace);
     }, error => {
-      this.alertService.danger(error.error.message);
+        this.snackBar.open(error.error.error.message, 'Close', { duration: 2000 });
```

Comment:
Extra 'error' here

Reply:
@becky-intelletive  Thanks for code reviewing this! Testing every spot will be tough.